src/STAT_FrontEnd.C

```diff
@@ -961,19 +624,12 @@ StatError_t STAT_FrontEnd::launchMrnetTree(StatTopology_t topologyType, char *to
     } /* if (applicationOption_ == STAT_SERIAL_ATTACH || applicationOption_ == STAT_SERIAL_GDB_ATTACH) */
     else
     {
-        if (lmonRmInfo_.rm_supported_types[lmonRmInfo_.index_to_cur_instance] == RC_alps)
```

Comment:
This block is now in STAT_lmonFrontEnd::createMRNetNetwork, but it doesn't have the CRAY_ALPS_APRUN_PID or CRAY_ALPS_STAGE_FILES logic, I put that logic in the STAT_ctiFrontEnd version assuming it's Cray specific.